scripts/quests/bastok/A_Question_of_Faith.lua

```diff
@@ -88,9 +88,10 @@ quest.sections =
                         player:hasKeyItem(xi.ki.DAWN_TALISMAN) and
                         quest:getVar(player, 'Prog') == 0
                     then
-                        npc:messageText(npc, 7733)
-                        npc:messageText(npc, 7746)
-                        npcUtil.popFromQM(player, npc, 16822456, {claim = true, hide = 0})
+                        if npcUtil.popFromQM(player, npc, ID.mob.BUGALLUG, {claim = true, hide = 0}) then
+                            npc:messageText(npc, ID.text.ALTANA_DIE)
```

Comment:
will need to rerun the quest on local to verify this works correctly
messageText() and message() doesnt look like is uses the npc param

after that i can approve if a full run through works